src/wasm/types.rs

```diff
@@ -38,6 +38,11 @@ type BtcScriptConfigWithKeypath = {
   scriptConfig: BtcScriptConfig;
   keypath: Keypath;
 };
+type BtcSignMessageSignature = {
+  sig: Uint8Array,
+  recid: bigint,
+  electrum_sig65: Uint8Array,
```

Comment:
In JS/TS, camelCase is more common, and we use it in this project too. `electrumSig65`.

You can use `#[serde(rename_all = "camelCase")]` on the SignMessageSignature struct to see if that works (check other instances or the serde docs).